Gruntfile.js

```diff
@@ -46,6 +46,7 @@ module.exports = function(grunt) {
   grunt.loadNpmTasks('grunt-contrib-jshint');
   grunt.loadNpmTasks('grunt-contrib-nodeunit');
 
-  grunt.registerTask('default', ['jshint', 'nodeunit']);
+  grunt.registerTask('test', ['jshint', 'nodeunit']);
```

Comment:
Nothing wrong per se, but you can add the `htmllint` one by default, and have the test on for CI. It's more flexible this way (and a common practice).